### ISSUE TITLE
LOH detection

### ISSUE THREAD

**Issue by romanhaa** [WGLab/PennCNV#11]
It's me again :) I'm curious why the LOH detection is not a feature anymore and if it has ever been for Affymetrix chips. I feel like there is a real lack of software for this kind of analysis.

**Comment by kaichop** [comment]
You can use PennCNV for LOH analysis. Just need to make a simple change to
the HMM file (hhall.hmm), change the 100 to 0.

**Comment by romanhaa** [comment]
Oh, you mean in B1_mean? And then I run detect_cnv.pl with '--loh' ? Because it says that parameter is obsolete and might be removed in future releases.

**Comment by kaichop** [comment]
Yes.

**Comment by romanhaa** [comment]
In this scenario, what do the states mean? They again range from 0 to 3. Is it the CN at that LOH call or again an internal parameter?

**Comment by kaichop** [comment]
state4 means LOH. You should see a lot of state4 cn=2 calls.

**Comment by romanhaa** [comment]
Hmm, interesting, I only see the state, no column for CN. Will run it again without "-tabout" even though I don't think that should make a difference. Or do I have to see which CN the LOH calls correspond to from the previous CN analysis (without "-LOH")?

**Comment by kaichop** [comment]
You did not provide any detail in your question so I have no way of knowing
exactly what you are talking about. By default -tabout is not added.

**Comment by romanhaa** [comment]
Sorry, I will try again with some more detail. I first did the normal detect_cnv.pl analysis (meaning without adding "-loh"). In the output file, I had a column with 'state' and one with 'CN'. 

`chr4:94375102-94525333        numsnp=90     length=150,232     state2,cn=1     GSM...     startsnp=SNP_A-4290445 endsnp=CN_1099144`

However, when running the same script with '-loh -tabout' (and the modified hmm file), the output file only contains one column that could either be 'state' or 'CN'. Here is an example:

`8	13780778	13783440	4	GSM...	SNP_A-8503123	SNP_A-8349269`

When you said that I should see state4 cn=2 calls, I was expecting two columns. Now my question would have been, if the one column I have is the state, which I have to compare manually to the CN in that region. However, I already started another run without '-tabout' and it gives me the expected output:

`chr14:21686225-22053063       numsnp=318    length=366,839     state4,cn=2     GSM...     startsnp=CN_635983 endsnp=SNP_A-8656339`

So everything fine now, just looks to me like the CN is not included in the output when using '-tabout'.
